packages/devtools_app/lib/src/screens/inspector/diagnostics_node.dart

```diff
@@ -466,9 +470,9 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
   /// Instance object for the Dart value because much of the relevant
   /// information to display good visualizations of Flutter values is stored
   /// in properties not in fields.
-  Future<Map<String, InstanceRef>> get valueProperties async {
+  Future<Map<String, InstanceRef>?>? get valueProperties async {
     if (_valueProperties == null) {
-      if (propertyType == null || valueRef?.id == null) {
+      if (propertyType == null || valueRef.id == null) {
         _valueProperties = Future.value();
         return _valueProperties;
```

Comment:
this can be combined into one line `return _valueProperties = Future.value();` Same for below in this method

Reply:
done